src/components/urls-list/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useContext, useEffect } from "react";
+import { UrlsContext } from "../../contexts/urls-context";
+import Item from "./item";
+
+export default function UrlsLsit() {
+  const { urlsList, getAllUrls } = useContext(UrlsContext);
+
+  useEffect(() => {
+    (async() => getAllUrls())();
+  }, []);
```

Comment:
Since it's just a single call, `async/await` is not doing anything here
```suggestion
  useEffect(() => {
    get AllUrls()
  }, []);
```